scripts/tests/qml_tests.sh

```diff
@@ -39,7 +63,20 @@ qmltest_compile || die
 print Y "Running the QML tests..."
 qmltest_run || die
 
+if [[ "$GRCOV_FILENAME" ]]; then
+  printn Y "Generating temp coverage file for lottie unit tests..."  
+  _grcov_qml test_"$GRCOV_FILENAME"
+fi
+
 printn Y "Cleaning the existing project... "
 qmltest_cleanup || die
 
+if [[ "$GRCOV_FILENAME" ]]; then  
+  printn Y "merging temp files to qml_lcov.info... "
+  grcov -t lcov -o "$GRCOV_FILENAME" test_"$GRCOV_FILENAME" || die "merging temp files to final failed"
```

Comment:
Well... in this file, there is only 1 GRCOVFILE. Use it directly. Just change line 68 to pass $GRCOV_FILENAME and remove this block.

Reply:
good point